server/gw.go

```diff
@@ -0,0 +1,33 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+)
+
+type GRPCGateway struct {
```

Comment:
Can we have a documentation/comment on GRPCGateway type. Helpful for consumers.

Reply:
Added.